indico/modules/rb_new/util.py

```diff
@@ -49,3 +60,23 @@ def search_for_rooms(filters, only_available=False):
     else:
         rooms = query.all()
     return rooms
+
+
+def get_buildings(location=None):
```

Comment:
location arg can go away when reverting the change to the legacy get_buildings in the model